Assets/Editor/EditorTimeoutSwitch.cs

```diff
@@ -0,0 +1,36 @@
+
+namespace Shopify.Tests {
+    using NUnit.Framework;
+    using UnityEditor;
+    using UnityEngine.TestTools;
+
+    public class EditorTimeoutWaiter {
```

Comment:
@mikkoh I built this as a complement to `StoppableWaitForTime`... in the editor you can only yield null from a `UnityTest` so the normal stoppable waiter wont work. WDYT?

Reply:
Is there any way to know whether we're in play mode or editor mode? This way we could use the same class

Reply:
yea with compiler directives, the downside is we cant use `CustomYieldInstruction` so the interface is completely different for the two.